tests/scripts/run_performance.sh

```diff
@@ -53,6 +53,11 @@ run_perf_models_llm_javelin_multi_device() {
     local test_marker=$2
 
     env pytest models/demos/falcon7b/tests -m $test_marker
+
+    # Mistral8x7b weights and cached files
+    export MIXTRAL_CKPT_DIR=/mnt/MLPerf/tt_dnn-models/Mistral/Mixtral-8x7B-v0.1
```

Comment:
@tapspatel @TT-billteng I think we need to stop putting env vars in scripts like this.

Should we instruct Miguel to check for `os.environ["CI"] == "true"` and set these in the test itself?

Reply:
Thanks for the warning.
I will update the perf test with this.

Should we update all of our tests to include this verification?

Reply:
We want to provide something more robust for model test writers to use so it's cleaner and clearer that it's on CI.

But this is how we would check it's on CI.

Reply:
Just pushed a commit with this change for all tests. Re-running the 3 pipelines now.

Reply:
feel like this is a model cache and we should have this somewhere as a pythorn/pytest fixture